cmake/Dependencies.cmake

```diff
@@ -1805,6 +1805,7 @@ endif()
 #
 set(TEMP_BUILD_SHARED_LIBS ${BUILD_SHARED_LIBS})
 set(BUILD_SHARED_LIBS OFF CACHE BOOL "Build shared libs" FORCE)
+set(FMT_INSTALL ON CACHE BOOL "")
```

Comment:
Can you please explain why `FMT_INSTALL` is needed?

Reply:
removed